eternalcore-core/src/main/java/com/eternalcode/core/configuration/implementation/PluginConfiguration.java

```diff
@@ -74,6 +74,16 @@ public Duration teleportTime() {
         }
     }
 
+    @Description({" ", "# Teleport section"})
+    public Tp tp = new Tp();
+
+    @Contextual
+    public static class Tp {
```

Comment:
Takim nazwom mówimy stanowcze nie, proszę zmienić tą nazwę na np. Teleports/TeleportSection

Reply:
Możesz też poprawić w sekcji `Tpa`

Reply:
Tak w zasadzie to **Section** używacie tylko w tłumaczeniu bo cały PluginConfiguration tak wygląda 

![image](https://user-images.githubusercontent.com/50597234/235500594-99663e29-b9ed-4730-a63b-4c0edf3b42ee.png)

Reply:
Można chociaż nazwać teleports